tests/components/dynalite/test_switch.py

```diff
@@ -34,3 +37,19 @@ async def test_switch_setup(hass, mock_device):
             {ATTR_SERVICE: "turn_off", ATTR_METHOD: "async_turn_off"},
         ],
     )
+
+
+@pytest.mark.parametrize("saved_state, level", [("on", 1), ("off", 0)])
+async def test_switch_restore_state(hass, mock_device, saved_state, level):
```

Comment:
Improve the test to also check that the state of the switch is as expected.

Reply:
thanks. fixed